lib/Mojolicious/Command/Author/generate/secret.pm

```diff
@@ -0,0 +1,85 @@
+package Mojolicious::Command::Author::generate::secret;
+use Mojo::Base 'Mojolicious::Command';
+use Mojo::File qw(path);
+use Mojo::Util qw(urandom_token);
+
+has description => 'Generate secret';
+has usage       => sub { shift->extract_usage };
+
+sub run {
+  my ($self, $secret_file) = (shift, shift);
+
+  $secret_file //= path($self)->sibling("mojo.secret");
```

Comment:
This seems wrong? path($self) doesn't make sense, nor should it default to writing files into the Perl lib directory.

Reply:
This is fixed